src/styles/ConfirmDialog.module.scss

```diff
@@ -2,4 +2,18 @@
   .large-button:not(:first-of-type) {
     margin-top: 0.5em;
   }
+
+  @media only screen and (max-width: 500px) {
+    // Set width to 98% when screen width is less than 500px
+    // 500px is the default width of the component
+    width: 98%;
+  }
+}
+
+.PromptDialog {
+  @media only screen and (max-width: 500px) {
+    // Set width to 98% when screen width is less than 500px
+    // 500px is the default width of the component
+    width: 98%;
+  }
```

Comment:
I don't see this class being used anywhere?

Reply:
Yes, my bad. The PromptDialog component uses the ConfirmDialog component as well.